src/market.py

```diff
@@ -229,12 +229,15 @@ def _make_match_orders_transaction(self, limit: int) -> TransactionInstruction:
         )
         return match_order_inst(params)
 
-    def _send_transaction(self, transaction: Transaction, signers: List[Account]):
+    def _send_transaction(self, transaction: Transaction, signers: List[Account]) -> str:
```

Comment:
I don't particularly like having transaction sending in the market module. I think we should help the API user create instructions and have the user combine the necessary instructions and send it.

Reply:
discussed via discord, agree that we can expose two API, one only return the instruction and allow the user to send it. The other make and send the transaction.